secant/Mocked Dependencies/OnboardingMocks.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  OnboardingMocks.swift
+//  secantTests
+//
+//  Created by Francisco Gindre on 9/20/21.
+//
+
+import Foundation
+class OnboardingStepProviderBuilder {
```

Comment:
Does this need reference semantics?  If we removed the builder, could we use value semantics here?

Reply:
removed this!